commands/payment_channel_daemon_test.go

```diff
@@ -1,547 +1,620 @@
-	})
+	err = series.SetupGenesisNode(ctx, genesis, genesisMiner.Address, files.NewReaderFile(genesisMiner.Owner))
+	require.NoError(err)
 
-	t.Run("Notifies when channels not found", func(t *testing.T) {
-		t.Parallel()
+	NewNode := func() *fast.Filecoin {
+		p, err := env.NewProcess(ctx, localplugin.PluginName, options, fastenvOpts)
+		require.NoError(err)
 
-		payer, err := address.NewFromString(fixtures.TestAddresses[2])
+		err = series.InitAndStart(ctx, p)
 		require.NoError(err)
-		target, err := address.NewFromString(fixtures.TestAddresses[1])
+
+		err = series.Connect(ctx, genesis, p)
 		require.NoError(err)
 
-		eol := types.NewBlockHeight(20)
-		amt := types.NewAttoFILFromFIL(10000)
+		err = series.SendFilecoinDefaults(ctx, genesis, p, 10000)
+		require.NoError(err)
 
-		daemonTestWithPaymentChannel(t, &payer, &target, amt, eol, func(d *th.TestDaemon, channelID *types.ChannelID) {
-			ls := listChannelsAsStrs(d, &target)[0]
+		return p
+	}
 
-			assert.Equal("no channels", ls)
-		})
-	})
+	return env, genesis, NewNode
 }
 
-func TestPaymentChannelVoucherSuccess(t *testing.T) {
+func TestPaymentChannelCreateSuccess(t *testing.T) {
 	t.Parallel()
 	require := require.New(t)
 
-	payer, err := address.NewFromString(fixtures.TestAddresses[2])
-	require.NoError(err)
-	target, err := address.NewFromString(fixtures.TestAddresses[1])
-	require.NoError(err)
+	// This test should run in 20 block times
+	ctx, cancel := context.WithDeadline(context.Background(), time.Now().Add(20*series.GlobalSleepDelay))
+	defer cancel()
 
-	eol := types.NewBlockHeight(20)
-	amt := types.NewAttoFILFromFIL(10000)
+	// Get basic testing environment
+	env, _, NewNode := BasicFastSetup(t, ctx, fast.EnvironmentOpts{})
 
-	daemonTestWithPaymentChannel(t, &payer, &target, amt, eol, func(d *th.TestDaemon, channelID *types.ChannelID) {
-		assert := assert.New(t)
+	// Teardown after test ends
+	defer env.Teardown(ctx)
+
+	// Start test
+	targetDaemon := NewNode()
+	payerDaemon := NewNode()
 
-		voucher := mustCreateVoucher(t, d, channelID, types.NewAttoFILFromFIL(100), &payer)
+	addrs, err := targetDaemon.AddressLs(ctx)
+	require.NoError(err)
+	targetAddr := addrs[0]
 
-		assert.Equal(*types.NewAttoFILFromFIL(100), voucher.Amount)
+	addrs, err = payerDaemon.AddressLs(ctx)
+	require.NoError(err)
+	payerAddr := addrs[0]
+
+	channelExpiry := types.NewBlockHeight(20)
+	channelAmount := types.NewAttoFILFromFIL(1000)
+
+	WithPaymentChannel(t, ctx, payerDaemon, payerAddr, targetAddr, channelAmount, channelExpiry, func(chanid *types.ChannelID) {
+		require.NotNil(chanid)
 	})
 }
 
-func TestPaymentChannelRedeemSuccess(t *testing.T) {
-	t.Parallel()
+// WithPaymentChannel takes a filecoin daemon and constructs a payment channel to the `target` for the `amt` for `eol`, and calls fn with the channel id
+func WithPaymentChannel(t *testing.T, ctx context.Context, p *fast.Filecoin, from, target address.Address, amt *types.AttoFIL, eol *types.BlockHeight, fn func(chanid *types.ChannelID)) {
```

Comment:
This doesn't really need a callback function, it would just return the `ChannelID`. This is because we don't have the goroutine anymore. I kept this as a callback for explicit in updating tests and I wasn't sure of the exact pattern I wanted to use. Really, either way I don't think it's needed as we can always mine, and then wait.

I will say the callback does help break up the tests and I think makes it easier to read.